frontend/src/components/ProductDetail.tsx

```diff
@@ -0,0 +1,28 @@
+import Grid from "@material-ui/core/Grid";
+import { useProductDetailStyles } from "../styles/ProductDetailStyles";
+
+interface ProductProp {
```

Comment:
Great use of an interface for the props. A better naming suggestion would be `ProductDetailsProps`